app/src/main/java/com/github/se/stepquest/map/Map.kt

```diff
@@ -364,22 +393,19 @@ fun Map(locationViewModel: LocationViewModel) {
         stopRoute = {
           showProgression = false
           locationViewModel.onPause()
+          locationViewModel.isFollowingRoute.value = false
```

Comment:
It might be better to delete this lne, so when press finish button, the go back button is still there (now when press finish button the go back buttonwill dispeare). The user can see their created route, and when the user wants to go back to the default map, it can pree the go back button. It may be a better way to do, so I would suggest to delete this line here.